docs/user/rstudio/ide/guide/code/projects.qmd

```diff
@@ -2,27 +2,27 @@
 title: "RStudio Projects"
 ---
 
-Expert data scientists keep all the files associated with a given project together --- input data, scripts, analytical results, and figures. This is such a wise and common practice that RStudio has built-in support for this via Projects. -[R4DS](https://r4ds.hadley.nz/workflow-scripts.html#workflow-projects)
-
-RStudio Projects make it straightforward to divide your work into multiple contexts, each with their own working directory, workspace, history, and source documents.
-
-Created from [Support article](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects)
+Expert data scientists keep all the files associated with a given project together --- input data, scripts, analytical results, and figures. This is such a wise and common practice that RStudio has built-in support for this via Projects. RStudio Projects make it straightforward to divide your work into multiple contexts, each with their own working directory, workspace, history, and source documents.
 
 ## Creating Projects
 
-RStudio projects are associated with R working directories. You can create an RStudio project:
+RStudio projects are associated with R working directories. 
+
+RStudio projects can be created:
 
 -   In a brand new directory
 
 -   In an existing directory where you already have R code and data
 
 -   By cloning a version control (Git or Subversion) repository
 
-To create a new project in the RStudio, use the **Create Project** command (available on the Projects menu and on the global toolbar):
+To create a new project in RStudio, use the **Create Project** command (available on the Projects toolbar in the top right corner or on the global toolbar at the top left corner). 
```

Comment:
Change the wording to **New Project** command instead of **Create Project** command because that's how it's labeled in the IDE. Maybe add clearer direction on how to get there with the global toolbar eg. File > New Project.

Reply:
Changed